qml/pages/OverviewPage.qml

```diff
@@ -92,6 +92,12 @@ Page {
             tdLibWrapper.getRecentStickers();
             tdLibWrapper.getInstalledStickerSets();
             tdLibWrapper.getContacts();
+            tdLibWrapper.getUserPrivacySettingRules(TelegramAPI.SettingAllowChatInvites);
```

Comment:
It's a bit like the requests above, but: Do we actually need them at this point? Getting them on demand inside getUserPrivacySettingRule if they're not filled would be pretty nice. The "empty" result would then be updated as soon as the real result arrives, anyway.

More aggressive alternative approach: Do we even need to cache them at all?

Reply:
:) I'm probably over-pessimistic, I simply want to have the data ready when the user opens up the settings page without values flickering around when the page is opened. There's always a trade-off and - you know me - I tend to cache more....